index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Cinema Ticket</title>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto&display=swap" rel="stylesheet">
+  <link rel="stylesheet" href="style.css">
+</head>
+<body>
+  <div class="ticket-container">
+    <div class="ticket-main">
+      <div class="ticket-text">CINEMA TICKET</div>
+      <div class="ticket-details">
+        <div class="category">VIP</div>
+        <div class="price">$10</div>
+      </div>
+    </div>
+    <div class="seance-details">
+      <div class="hour">08:00 PM
+      </div>
+      <div class="cinema-room-details">
+        <div class="gate">GATE: 03</div>
+        <div class="seat"> SEAT: B43</div>
+      </div>
+      <div class="ticket-number">TICKET NO: 103752
+      </div>
```

Comment:
```suggestion
      <div class="ticket-number">TICKET NO: 103752</div>
```
This new line is not needed